content/en/docs/tasks/debug/debug-cluster/_index.md

```diff
@@ -14,6 +14,12 @@ problem you are experiencing. See
 the [application troubleshooting guide](/docs/tasks/debug/debug-application/) for tips on application debugging.
 You may also visit the [troubleshooting overview document](/docs/tasks/debug/) for more information.
 
+{{% alert title="Note" %}}
+If you are looking for troubleshooting
+{{<glossary_tooltip text="kubectl" term_id="kubectl">}} click
+[here](/docs/tasks/debug/debug-cluster/troubleshoot-kubectl-connectivity/).
```

Comment:
We don't favor using 'here' as the link word, please leave something meaningful as the link anchor.